canonical-web-and-design/jp.ubuntu.com
# Set width height to thumbnails

## Done

Fix blog article thumbnails. 

## QA

https://jp.ubuntu.com/blog
Check the thumbnails, you will notice they do not have the same size.
https://jp-ubuntu-com-242.demos.haus/blog
Check the thumbnails, you will notice they all have the same size(354x180) and cover the whole width of the block.

## Issue

Part of: https://github.com/canonical-web-and-design/canonicalwebteam.blog/pull/153

## Screenshot
Before:
![image](https://user-images.githubusercontent.com/6387619/94822260-5c4a1b80-03fa-11eb-856a-40301a574e0b.png)

Alfter:
![image](https://user-images.githubusercontent.com/6387619/94822423-8bf92380-03fa-11eb-9a7f-932dc8b88142.png)
